AsyncDisplayKit/ASVisibilityProtocols.m

```diff
@@ -0,0 +1,23 @@
+//
+//  ASVisibilityProtocols.m
+//  Pods
+//
+//  Created by Garrett Moon on 4/28/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import "ASVisibilityProtocols.h"
+
+ASLayoutRangeMode ASLayoutRangeModeForVisibilityDepth(NSUInteger visibilityDepth)
```

Comment:
There's a difference between range modes and visibility distance. For example, in high memory pressure environments, you might want to map these differently. Or perhaps the other way around. Anyway, I felt the distinction was important – this measures the number of actions to get a view controller visible. Range modes specify a bunch of different things. They're related but different.